src/main/java/business/Helper/LocalDateTimeParser.java

```diff
@@ -0,0 +1,17 @@
+package business.Helper;
+
+import java.time.LocalDateTime;
+
+public class LocalDateTimeParser {
+    // example dateTimeStr:
+    // 2 0 2 1 - 0 7 - 1 1 T  1  6  :  4  4  :  0  0 +07:00
+    // 0 1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 18
+    public static LocalDateTime parse(String time){
```

Comment:
Nice! Java already had built-in APIs that help us convert Strings into LocalDateTime

if you have time, see https://www.java67.com/2016/04/how-to-convert-string-to-localdatetime-in-java8-example.html